### PR TITLE
Feat/backend response format 

### PR DESCRIPTION
**Pull request by Druidman**
Changes:
-Response format: Changed response format in /api/quiz/ endpoint
-Docs: Added response description in documentation

### CODE REVIEW

File: backend/run.py
Comment (Kumala3) [inline backend/run.py:1]: Why did you delete the check of file name with being **main**?

**Review by Kumala3 — COMMENTED** [review]
As I can see you haven't implemented the refactoring yet, don't forget to handle the situation when I don't send an id as a argument (query), it's crucial to return an error with appropriate error_code like 400 (bad request), and message_error explaining that the required id param is missing

File: backend/run.py
Comment (Druidman) [reply backend/run.py:1]: bo jest niepotrzebny
